synapse/models/orgs.py

```diff
@@ -19,6 +21,62 @@ def seedOrgAlias(self, prop, valu, **props):
             node = self.core.formTufoByProp('ou:org', guid(), alias=valu, **props)
         return node
 
+    @modelrev('ou', 201802281621)
+    def _revModl201802281621(self):
+        '''
+        Combine ou:has* into ou:has
```

Comment:
@invisig0th - do you want this to be `ou:org:has` instead of `ou:has`?

Reply:
yea, lets move to the new convention for consistency